tests/Doctrine/Tests/ORM/Functional/Ticket/GH7820Test.php

```diff
@@ -73,6 +78,41 @@ public function testWillFindSongsInPaginator() : void
             }, iterator_to_array(new Paginator($query)))
         );
     }
+
+    /** @group GH7837 */
+    public function testWillFindSongsInPaginatorEvenWithCachedQueryParsing() : void
+    {
+        $cache = $this->_em->getConfiguration()
+            ->getQueryCacheImpl();
+
+        assert($cache instanceof ClearableCache);
+
+        $cache->deleteAll();
```

Comment:
this is not needed

Reply:
Sadly needed, otherwise the two tests interfere with each other

Reply:
which ones? Running whole GH7820Test class runs fine for me without this statement

Reply:
After another look, I can confirm this is neeed. Otherwise removing the fix would cause failure on first assertion